hokusai/staging.yml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: rosalind-web
+  namespace: default
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: rosalind
+        component: web
+        layer: application
+      name: rosalind-web
+    spec:
+      containers:
+        - name: rosalind-web
+          env:
+            - name: PORT
+              value: "8080"
+            - name: RAILS_ENV
+              value: production
+            - name: RAILS_SERVE_STATIC_FILES
+              value: 'true'
+            - name: RAILS_LOG_TO_STDOUT
+              value: 'true'
+            - name: PUMA_BIND
```

Comment:
Was this the key to getting puma starting?

Reply:
Not sure about that — this was updated bc I grabbed the latest template files from https://github.com/artsy/artsy-hokusai-templates/tree/master/rails-puma

I think the comment above was the real issue, though there may have been more than one.

Reply:
This was the problem I saw in Reflection with Puma only binding to localhost